### PR TITLE
Add Community Support Working Group charter

### PR DESCRIPTION
**Pull request by tdcmeehan**

### CODE REVIEW

**Review by highker — APPROVED** [review]

File: working-groups/community-support-wg.md
Comment (yingsu00) [inline working-groups/community-support-wg.md:41]: Shall we use the current TSC meeting to discuss the WG related items? I feel it will take more effort and time to organize another meeting for both organizers and attendees.

**Review by zhenxiao — APPROVED** [review]

File: working-groups/community-support-wg.md
Comment (tdcmeehan) [reply working-groups/community-support-wg.md:41]: We could consider that, but I'm not a fan because:

* TSC meetings are monthly, and I think this group should meet much more frequently to make a positive impact in the community.
* If members are already constrained on time, then they should not join this group.  People in this group need to have buy in from their current employer that they can carve out dedicated time to supporting the community.
* This gives people who are not in the TSC a way to participate in the community.  I.e. this is another role for people in the community to participate in, if they are not already a committer, TSC member, a member of the conformance program, or the outreach committee.

Comment (yingsu00) [reply working-groups/community-support-wg.md:41]: Makes sense. Thanks for your ideas.

**Review by yingsu00 — APPROVED** [review]

**Review by rongrong — APPROVED** [review]

**Comment by tdcmeehan** [comment]
Made some minor edits meant to encourage frank discussions and free thought.  Will merge tomorrow unless I hear any objections.

**Pull request merged**
